pom.xml

```
@@ -2120,6 +2120,23 @@
                 </rules>
               </configuration>
             </execution>
+            <execution>
+              <id>ban-jetty-stringutil</id>
+              <goals>
+                <goal>enforce</goal>
+              </goals>
+              <phase>process-sources</phase>
+              <configuration>
+                <rules>
+                  <restrictImports>
+                    <reason>Jetty's StringUtil is disabled by default.  Replace it with Apache Common's StringUtils.</reason>
```

"disabled by default" only applies to annotation processing.

```suggestion
                    <reason>Use similar class from Apache Commons Lang</reason>
```

---

Thanks!
It will be modified in the next commit.